lib/toggleClass.js

```diff
@@ -0,0 +1,9 @@
+function toggleClass(el, value) {
+  if (el.classList.contains(value)) {
```

Comment:
Using classList is fine, but I would love to see it use the built in methods in case we make some customizations like for an array of classes.